README.md

```diff
@@ -177,7 +177,7 @@ engine_base = compile_model(
     model="zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/base-none", 
     batch_size=batch_size,
 )
-benchmarks_base = engine_base.benchmark(sample_inputs)
+benchmarks_base = engine_base.benchmark_bert(sample_inputs)
```

Comment:
This is the subsection for ResNet50. Did you mean to have a different one for BERT?

Reply:
will revert these